### PR TITLE
Accessible screen titles

### PR DESCRIPTION
**Pull request by AndriiHorishniiMOC**
Accessible screen titles

https://glia.atlassian.net/browse/MOB-1278

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java [hunk 1]
```diff
@@ -43,6 +43,7 @@ public class CallActivity extends AppCompatActivity {
         navigateToChat();
         finish();
     };
+    private final CallView.OnTitleUpdatedListener onTitleUpdatedListener = this::setTitle;
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java:46]: no reason to keep this callback global, you can just inline it. `callView.setOnTitleUpdatedListener(this::setTitle);`

File: widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java [hunk 2]
```diff
@@ -71,6 +73,13 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         if (savedInstanceState == null) {
             startCallWithPermissions();
         }
+
+        // Set title for accessibility screen reader
+        setTitle(
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java:78]: please refactor this into a separate method to keep `onCreate` cleaner. Method name can replace the need for comments as well.

**Review by jakob-mc — CHANGES_REQUESTED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java (hunk 1 above)
Comment (AndriiHorishniiMOC) [reply widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java]: done

File: widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java (hunk 2 above)
Comment (AndriiHorishniiMOC) [reply widgetssdk/src/main/java/com/glia/widgets/call/CallActivity.java:80]: done

File: widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java [hunk 3]
```diff
@@ -44,6 +45,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         ChatHeadLayout chatHeadLayout = findViewById(R.id.chat_head_layout);
         chatHeadLayout.setIsChatView(true);
 
+        chatView.setOnTitleUpdatedListener(onTitleUpdatedListener);
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java:48]: inline here as  well

File: widgetssdk/src/main/java/com/glia/widgets/survey/SurveyActivity.java [hunk 4]
```diff
@@ -24,7 +25,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void prepareSurveyView() {
         surveyView = findViewById(R.id.survey_view);
-        surveyView.setCallback(this);
+        surveyView.setOnTitleUpdatedListener(onTitleUpdatedListener);
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/survey/SurveyActivity.java:28]: and here

**Review by jakob-mc — CHANGES_REQUESTED** [review]

**Review by jakob-mc — APPROVED** [review]

**Pull request merged**